app/components/download-switch/download-switch.tsx

```diff
@@ -154,8 +154,8 @@ export function DownloadSwitch({
   const [progressPercentages] = useState<Map<string, number>>(
     () => new Map(tracks.map((track) => [track.trackId, 0])),
   )
-  const [resumableDownloads] = useState<Map<string, DownloadResumable | null>>(
-    () => new Map(tracks.map((track) => [track.trackId, null])),
+  const [resumableDownloads] = useState<Map<string, DownloadResumable | undefined>>(
```

Comment:
With the `const download = resumableDownloads.get(...)` later in the file we ended up with `download: null | undefined | DownloadResumable`, so I switched to `undefined` here to avoid checking for both nullable values.